pkg/kubelet/metrics/metrics.go

```diff
@@ -47,6 +47,9 @@ const (
 	// Metrics keys of device plugin operations
 	DevicePluginRegistrationCountKey = "device_plugin_registration_count"
 	DevicePluginAllocationLatencyKey = "device_plugin_alloc_latency_microseconds"
+	// Metrics keys of volume operations
+	VolumeManagerOpeationsKey      = "volumemanager_operation_duration_seconds"
+	VolumeManagerOpeationsErrorKey = "volumemanager_operation_errors_total"
```

Comment:
Replaced by `volumeManagerOpeationsKey` and `volumeManagerOpeationsErrorKey`? As they are all internal variables.